pkg/rancher-components/src/components/Pill/RcTag/RcTag.vue

```diff
@@ -2,14 +2,16 @@
 import RcButton from '@components/RcButton/RcButton.vue';
 import { RcTagProps } from './types';
 
-const props = withDefaults(defineProps<RcTagProps>(), { disabled: false, showClose: false });
+const props = withDefaults(defineProps<RcTagProps>(), {
+  disabled: false, showClose: false, highlight: undefined
+});
 const emit = defineEmits(['close']);
 </script>
 
 <template>
   <div
     class="rc-tag"
-    :class="{[props.type]: true, disabled: props.disabled}"
```

Comment:
For accessibility when focus/hover falls off a tag but we still need to indicate the tag is a part of the current interaction we still need to be able to highlight.